crates/json-abi/src/item.rs

```diff
@@ -563,6 +574,12 @@ impl Event {
         event_signature(&self.name, &self.inputs)
     }
 
+    /// Returns this event's full signature: `event $name($($inputs indexed names),*)`.
```

Comment:
Can you also add the paragraph from Function's `full_signature`?

Reply:
will do